joomla/joomla-cms
# Custom field display style

Pull Request for Issue #31385 .

Summary of Changes

Added style to _legacy.scss to remove bullets and indent from list of custom fields.

This arises from sense of frustration experienced by users migrating from Joomla 3 to Joomla 4 where custom fields that were set as definition list items in the former are set as unordered list items in the latter.

Testing Instructions

Apply the patch and run npm ci to recompile the css files
Create some custom fields and populate them for an article.
View the article in the front end.

Actual result BEFORE applying this Pull Request

![image](https://user-images.githubusercontent.com/1560390/99094994-774ca580-25cc-11eb-8390-48931e6c22e6.png)

Expected result AFTER applying this Pull Request

![image](https://user-images.githubusercontent.com/1560390/99095033-829fd100-25cc-11eb-89c4-5cfa050f40fe.png)

Documentation Changes Required

Some comment in Help file - I will fix after merge.
